ketcher-autotests/tests/User-Interface/Editing-Tools/3D-Viewer/three-d-viewer.spec.ts

```diff
@@ -0,0 +1,264 @@
+  });
+
+  test('Structure with Aromatic Bonds without Save Position', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-1943
+    Description: 3D window is opened. The structure with a circle inside the cycle is displayed in the window. 
+    The structure isn't changed.
+    */
+    await openFileAndAddToCanvas('benzene-with-aromatic-bonds.mol', page);
+    await selectTopPanelButton(TopPanelButton.ThreeD, page);
+    await delay(DELAY_IN_SECONDS.THREE);
+    await moveMouseToTheMiddleOfTheScreen(page);
+    const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+    await dragMouseTo(x - 20, y, page);
+    await pressButton(page, 'Cancel');
+  });
+});
+
+test.describe('3D Viewer', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test('Button and tooltip verification', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1931
+    Description: '3D Viewer' button is present on the main toolbar.
+    */
+    await selectTopPanelButton(TopPanelButton.ThreeD, page);
+    await delay(DELAY_IN_SECONDS.THREE);
+    await expect(page).toHaveScreenshot({
+      animations: 'disabled',
+      maxDiffPixelRatio: 0.05,
+    });
+  });
+
+  test('Empty canvas', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1932
+    Description: 3D Viewer window is opened. There is no structure in the opened  window.
+    */
+    await selectTopPanelButton(TopPanelButton.ThreeD, page);
+    await expect(page).toHaveScreenshot({
+      animations: 'disabled',
+      maxDiffPixelRatio: 0.05,
+    });
+  });
+
+  test('One simple structure', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1933
+    Description: A structure (e.g. Benzene) is created on the canvas.
+    3D window is opened. Benzene is drawn in it. 
+    */
+    await selectRing(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await selectTopPanelButton(TopPanelButton.ThreeD, page);
+    await expect(page).toHaveScreenshot({
+      animations: 'disabled',
+      maxDiffPixelRatio: 0.05,
+    });
+  });
+
+  test('3D Rotation of the simple structure with Save Position', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-1935
+    Description: 3D window is opened.
+    Benzene is drawn in it. The structure is spinned.
+    Position of the structure on the canvas is changed. 
+    */
+    await selectRing(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    const initialStructureData = await getKet(page);
+    await selectTopPanelButton(TopPanelButton.ThreeD, page);
+    await moveMouseToTheMiddleOfTheScreen(page);
+    const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+    await dragMouseTo(x, y - 30, page);
+    await pressButton(page, 'Apply');
+
+    // Get the structure data after making changes
+    const changedStructureData = await getKet(page);
+
+    // Compare the initial and changed structure data
+    if (
```

Comment:
I propose to use usual `expect` syntax. You can check the following links:
https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-strict-equal
https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-equal
https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-match-object

Reply:
Done